src/components/EReceipt.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import ONYXKEYS from '../ONYXKEYS';
+import * as StyleUtils from '../styles/StyleUtils';
+import transactionPropTypes from './transactionPropTypes';
+import styles from '../styles/styles';
+import * as Expensicons from './Icon/Expensicons';
+import Icon from './Icon';
+import Text from './Text';
+import * as ReportUtils from '../libs/ReportUtils';
+import * as CurrencyUtils from '../libs/CurrencyUtils';
+import * as CardUtils from '../libs/CardUtils';
+import variables from '../styles/variables';
+import useLocalize from '../hooks/useLocalize';
+import EReceiptThumbnail from './EReceiptThumbnail';
+
+const propTypes = {
+    /* transactionID */
+    transactionID: PropTypes.string.isRequired,
+
+    /* Onyx Props */
```

Comment:
Could this be more specific?

Reply:
Why do we need both transaction and transactionID as separate props? The transactionID should be in transaction, right?

Reply:
it's an ONYX prop, this is populated from onyx